test/functional/wallet_descriptor.py

```diff
@@ -16,18 +16,22 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
         self.extra_args = [['-keypool=100']]
+        self.wallet_names = []
```

Comment:
Why did this need to be added? I'm not seeing it used anywhere

Reply:
To ensure that no default wallets are created.